dev-libs/ferrisloki/ferrisloki-3.0.13-r3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic autotools
+
+DESCRIPTION="Loki C++ library from Modern C++ Design"
+HOMEPAGE="http://www.libferris.com/"
+SRC_URI="mirror://sourceforge/witme/${P}.tar.bz2"
```

Comment:
While at it, could you also update `HOMEPAGE`?
```
dev-libs/ferrisloki
  HttpsUrlAvailable: version 3.0.13-r3: HOMEPAGE: HTTPS url available: http://www.libferris.com/ -> https://data-room-software.org/libferris/
```

Reply:
Done, thanks.